src/modules/ticket/reducer.ts

```diff
@@ -0,0 +1,89 @@
+import {Alert} from 'react-native';
+import {Action} from 'redux';
+import {t} from '@translations';
+
+export const BUY_TICKET_REQUEST = 'BUY_TICKET_REQUEST';
```

Comment:
Personally I would put these 3 constants + the BuyTicketRequestAction, BuyTicketSuccessAction, BuyTicketFailAction and TicketAction definitions in the actionCreators.ts, but it really depends on the applicable best practices at a company or the coding style. I may be wrong :))